resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -173,7 +173,7 @@ function task_install {
     ln -s "../apps/${real_name}/bin/<%= basename %>" "${DESTDIR}${symbindir}/<%= basename %>"
     ln -s "../server/apps/${real_name}/bin/<%= basename %>" "${DESTDIR}${uxbindir}/<%= basename %>"
 <% end -%>
-    install -d -m 0755 "${DESTDIR}${rundir}"
+    install -d -m 0755 -o <%= EZBake::Config[:user] %> -g <%= EZBake::Config[:group] %> "${DESTDIR}${rundir}"
```

Comment:
I was trying to find something similar for debian, but I'm not coming up with anything concrete. The closes thing debian has is to be able to specify directories (https://www.debian.org/doc/manuals/maint-guide/dother.en.html#dirs), but I don't see where we would be able to specify permissions